### ISSUE TITLE
[TODO][V2]: Decide whether to keep the limitation of using a separate validation set without a separate test set

### ISSUE THREAD

**Issue by hwpang** [chemprop/chemprop#639]
**Notes**
At https://github.com/chemprop/chemprop/blob/v2/dev/chemprop/cli/train.py#L580, we currently limit it such that if a separate validation set is given (`val_path`), then a separate test set (`test_path`) also needs to be given. Note that the error message `'val_path' must be specified if 'test_path' is provided!` is currently incorrect and will be fixed in https://github.com/chemprop/chemprop/pull/608.

This limitation was not in v1, and we should discuss if we should keep this limitation and why.

**Comment by kevingreenman** [comment]
superseded by #645
